Patches/Vanilla Apparel Expanded/Patch_Apparel_Industrial.xml

```diff
@@ -87,7 +87,7 @@
 				<value>
 					<Bulk>5</Bulk>
 					<WornBulk>1</WornBulk>
-					<ArmorRating_Sharp>1.2</ArmorRating_Sharp>
+					<ArmorRating_Sharp>0.8</ArmorRating_Sharp>
```

Comment:
This still seems somewhat high. It might be appropriate it it's some kind of specialized, extra protective gear, but I'd like you to take a glance at it to make sure that was your intention. Might as well take a look at blunt while you're at it. 

For reference from base CE:
Cloth: 0.01 sharp multiplier, 0.015 blunt multiplier
Plain Leather: 0.05 sharp multiplier, 0.04 blunt multiplier

Parka: 10mm thick (StuffEffectMultiplierArmor)
Jacket: 5mm thick (StuffEffectMultiplierArmor)

Armor = StuffEffectMultiplierArmor * armor multiplier

Reply:
Changed the armor values of the clothing in question. I'm waiting for your feedback now.

Reply:
Alright, looks good.